### PR TITLE
remove un-implemented function from docs

### PR DESCRIPTION
**Pull request by awang**

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -101,7 +101,6 @@ small example that shows adding a custom span to a Flask application::
         # Or just trace part of a function with the `trace`
         # context manager.
         with tracer.trace("thumbnails.save") as span:
-            span.set_meta("thumbnails.sizes", str(sizes))
```
Comment (palazzem) [inline docs/index.rst]: do we want to use a `set_tag` instead? just to keep the example complete

Comment (awang) [inline docs/index.rst]: sure -- what did you have in mind exactly?

Comment (LotharSee) [inline docs/index.rst]: This method is implemented and works well.
https://github.com/DataDog/dd-trace-py/blob/6909a97184e00fc8e110a309230882a58d072dcb/ddtrace/span.py#L148

Comment (awang) [inline docs/index.rst]: updated, i removed the wrong function

**Pull request merged**